euth/contrib/base_models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from django.utils import timezone
+
+class TimeStampedModel(models.Model):
+
+    created = models.DateTimeField(editable=False, default=timezone.now)
+    modified = models.DateTimeField(blank=True, null=True, editable=False)
+
+    class Meta:
+        abstract = True
+
+    def save(self, *args, **kwargs):
+        if not self.pk is None:
```

Comment:
This expression is hard to read and understand. I would go for one of these:

1. `if self.pk`
2. `if self.pk is not None`